playbook/build_app_ami/main.yml

```diff
@@ -24,4 +24,4 @@
     - ../../roles/app_install
     - ../../roles/env_config
     - ../../roles/semanage_apps
-    - ../../roles/semanage_virtualenv
+    #- ../../roles/semanage_virtualenv
```

Comment:
if this is no longer used can it and the files it points to be removed?

Reply:
Was thinking of removing that later, after testing it, along with some debug messages. However, I'm not needing those debug messages anymore so will remove them. Will help not to forget also :-)

Reply:
Was thinking of removing that later, after testing it, along with some debug messages. However, I'm not needing those debug messages anymore so will remove them. Will help not to forget also :-)